CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## v0.19.0 (2018-2-15)
 
+### TestCafe Live: See instant feedback when working on tests
```

Comment:
Shall we add a section about `TestCafe Live` in our [`Recipes`]?(https://devexpress.github.io/testcafe/documentation/recipes/)

Reply:
I don't know. I guess we'll add some mention about it to the landing page and `readme.md`. We need to discuss this